src/main/java/org/mifos/processor/bulk/schema/BatchDTO.java

```diff
@@ -5,73 +5,97 @@
 
 public class BatchDTO {
 
-    private String batchId;
+    private String batch_id;
```

Comment:
why do we have the same file with a different naming convention for variables? some in camel case and some in snake case?

Reply:
everything is followed as per the existing code.

Reply:
New code should be better than existing code